nixos/doc/manual/development/replace-modules.xml

```diff
@@ -0,0 +1,36 @@
+<section xmlns="http://docbook.org/ns/docbook"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        xmlns:xi="http://www.w3.org/2001/XInclude"
+        version="5.0"
+        xml:id="sec-replace-modules">
+
+<title>Replace Modules</title>
+
+<para>It's also possible to disable modules allowing the
+  implementation to be replaced.  This can be used to import a set of
+  modules from another channel while keeping the rest of the system
+  on a stable release.</para>
+
+<para><literal>disabledModules</literal> is a top level attribute like
+  <literal>options</literal> and <literal>config</literal>.  It
+  contains a list of modules that will be disabled. This can either be
+  a string with the filename relative to &lt;nixpkgs/nixos/modules&gt;
+  or the full path to the module.</para>
+
+<programlisting>
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  disabledModules = [ "services/databases/postgresql.nix" ];
+
+  options = {
+    services.postgresql.enable = mkOption {
+      description = "Replaced option.";
```

Comment:
This example is not working as an option should requires more information to be listed.